compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Markdown.hs

```diff
@@ -211,8 +211,11 @@ fct2md :: FunctionDoc -> RenderOut
 fct2md FunctionDoc{..} = mconcat
     [ renderAnchorInfix "" fct_anchor $ T.concat
         [ "**", escapeMd $ unFieldname fct_name, "**  " ]
-    , renderLinesDep $ \env ->
-        maybe [] (\t -> ["&nbsp; : " <> type2md env t]) fct_type
+    , renderLineDep $ \env -> T.concat
+        [ "&nbsp; : "
```

Comment:
What is the rational for having a `&nbsp;` followed by a regular space? This seems odd to me.

Reply:
Markdown would interpret two spaces in a weird way, and `&nbsp; ` is enough to get HTML to accept this as a significant whitespace (although `&nbsp;&nbsp;` would work just as well).

(But in my new PR I get rid of this anyway and start using blockquotes everywhere.)

Reply:
Thanks for the explanation.